doc/api/process.md

```diff
@@ -1188,6 +1188,15 @@ console.log(process.getgroups());         // [ 27, 30, 46, 1000 ]
 *Note*: This function is only available on POSIX platforms (i.e. not Windows
 or Android).
 
+## process.isExiting
+<!-- YAML
+added: vX.X.X
```

Comment:
`vX.X.X` -> `REPLACEME`

Reply:
Aside: Maybe some of the cool new `md` linting can get a custom rule to validate these YAML bits? /cc @watilde